### DISCUSSION TITLE
Database error since 3.0.7

### DISCUSSION THREAD

**Discussion by Kalilovisky** [Brewtarget/brewtarget#778] [General]
Dear Matt,

I continue to experience issues with database imports since version 3.0.7.

To replicate this problem, simply create a recipe in version 3.0.7 and attempt to copy it to version 3.0.8 or 3.0.9 (changing the data or backup directory).

Is there any possibility of this issue being resolved in version 3.0.10?

Thank you.

**Comment by matty0ung** [comment]
Moving to issue https://github.com/Brewtarget/brewtarget/issues/780
